packages/react-components/src/components/VideoTile.tsx

```diff
@@ -129,18 +133,38 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
           )}
         </Stack>
       )}
-      {displayName && showDisplayName && (
-        <Text
-          className={mergeStyles(
-            isVideoReady ? videoHint : disabledVideoHint,
-            // when video is on, the displayName has a grey-ish background, so no use of theme
-            { color: isVideoReady ? palette.neutralPrimary : theme.palette.neutralPrimary },
-            styles?.displayNameContainer
+
+      <Stack
+        horizontal
+        className={mergeStyles(
+          isVideoReady ? videoHint : disabledVideoHint,
+          // when video is on, the displayName has a grey-ish background, so no use of theme
+          { color: isVideoReady ? palette.neutralPrimary : theme.palette.neutralPrimary },
+          styles?.displayNameContainer
+        )}
+      >
+        <Stack.Item>
+          {displayName && (
+            <Text
+              className={mergeStyles(displayNameStyle, {
+                color: isVideoReady ? palette.neutralPrimary : theme.palette.neutralPrimary
+              })}
+            >
+              {displayName}
+            </Text>
           )}
-        >
-          {displayName}
-        </Text>
-      )}
+        </Stack.Item>
+        <Stack.Item>
+          {showMuteIndicator && isMuted !== undefined ? (
+            isMuted ? (
+              <MicOffIcon size="small" />
+            ) : (
+              <MicIcon size="small" />
+            )
+          ) : null}
```

Comment:
cant we just say { a && b && <Component/> } we dont need to else return null?

Reply:
Sure.